src/pages/Organization/AdminLogin.tsx

```diff
@@ -35,53 +40,53 @@ function AdminLogin() {
+        <div className="md:w-3/5 md:p-5 sm:w-full sm:p-2 dark:bg-dark-frame-bg dark:rounded-none">
           <div className="py-10 sm:py-8 ">
             <h2 className="text-2xl font-bold text-primary dark:text-dark-text-fill ">
               {t('Sign in using')}
             </h2>
-            <div className="border-2 w-10 bg-primary border-primary inline-block mb-2" />
+            <div className="inline-block w-10 mb-2 border-2 bg-primary border-primary" />
             <div className="flex justify-center my-2">
               <a
                 href="#link"
-                className="border-2 border-gray-200 rounded-full p-3 mx-1"
+                className="p-3 mx-1 border-2 border-gray-200 rounded-full"
               >
                 <FaGoogle className="text-sm dark:text-white" />
               </a>
             </div>
-            <p className="text-gray-400 my-3 dark:text-dark-text-fill ">
+            <p className="my-3 text-gray-400 dark:text-dark-text-fill ">
               {t('or use your email account')}
             </p>
             <div className="flex flex-col items-center">
               <form action="#none" onSubmit={handleSubmit(onSubmit)}>
-                <div className="bg-gray-100 w-64 p-2 flex items-center mb-2 dark:bg-dark-bg ">
-                  <FaRegEnvelope className="text-gray-400 mr-2" />
+                <div className="flex items-center w-64 p-2 mb-2 bg-gray-100 dark:bg-dark-bg ">
+                  <FaRegEnvelope className="mr-2 text-gray-400" />
                   <input
                     type="email"
                     {...register('email', { required: 'Email is required' })}
                     placeholder={t('Email')}
-                    className="bg-gray-100 outline-none text-sm flex-1 text-gray-400 dark:border-white dark:bg-dark-bg dark:text-white "
+                    className="flex-1 text-sm text-gray-400 bg-gray-100 outline-none dark:border-white dark:bg-dark-bg dark:text-white "
                   />
                 </div>
-                <div className="text-left mb-1">
+                <div className="mb-1 text-left">
                   {errors.email && (
                     <small className="text-red-600">
                       {errors.email.message}
                     </small>
                   )}
                 </div>
 
-                <div className="bg-gray-100 w-64 p-2 flex items-center rounded mb-2 dark:border-white dark:bg-dark-bg">
-                  <MdLockOutline className="text-gray-400 mr-2 " />
+                <div className="flex items-center w-64 p-2 mb-2 bg-gray-100 rounded dark:border-white dark:bg-dark-bg">
+                  <MdLockOutline className="mr-2 text-gray-400 " />
                   <input
                     type={passwordShown ? 'text' : 'password'}
                     {...register('password', {
                       required: 'Password is required',
```

Comment:
On the password field, an error is only thrown when the field is left empty. Please add more validations like the minimum number of characters, having atleast a capital letter, a number and a special character.

Reply:
Hello Shyaka, on the login form, we don't validate the length since it would be possible to create a weak password on the server side which in turn would prevent you from logging in. The validation only works during the registration but is not suitable here